src/Animator.elm

```diff
@@ -243,14 +243,14 @@ type alias Timeline state =
 
 So, if you previously had a `Bool` in your model:
 
-    { checked = Bool }
+    type alias Model = { checked : Animator.Timeline Bool }
```

Comment:
Nice!  Could you change this one to 

```
type alias Model = { checked : Bool }
```
?

Reply:
Oops :open_mouth:
Yes, it's done :+1: